packages/stark-ui/src/modules/minimap/components/minimap.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="stark-minimap">
+	<div class="stark-minimap-dots">
+		<ul>
+			<li *ngFor="let item of items; trackBy: $index"
+				[ngClass]="{'selected':isItemVisible(item)}">
+				<i></i>
+			</li>
+		</ul>
+	</div>
+	<div class="stark-minimap-dropdown-toggle"
+		 [ngClass]="{'visible':isDisplayedMenu}">
+		<button mat-button class="md-icon-button stark-minimap-dropdown-toggle-button"
+				(click)="toggleDropdownMenu()" 
+				aria-label="Toggle columns"
+				matTooltip="'STARK.TABLE.TOGGLE_COLUMNS'"
+				[matTooltipPosition]="below"
+				translate>
+			<mat-icon class="down" svgIcon="menu-down"></mat-icon>
+			<mat-icon class="up" svgIcon="menu-up"></mat-icon>
+		</button>
+		<div class="stark-minimap-dropdown-toggle-menu">
+			<div class="stark-minimap-dropdown">
+				<mat-checkbox *ngFor="let item of items; trackBy : $index"
```

Comment:
Currently, Travis is failing because of this: `$index`.... I think you should put a function here instead